### PR TITLE
move to g3log

### PR DESCRIPTION
**Pull request by KjellKod**

### CODE REVIEW

**Comment by craig-cogdill** [comment]
:+1:

File: src/Crowbar.cpp [hunk 1]
```diff
@@ -4,8 +4,9 @@
 #include <zframe.h>
 
 #include "Crowbar.h"
-#include "boost/thread.hpp"
-#include "g2log.hpp"
+#include <boost/thread.hpp>
+#include <g3log/g3log.hpp>
+#include <Death.h>
```
Comment (craig-cogdill) [inline src/Crowbar.cpp]: To remove. John's pull request took the register for death event out of Crowbar

File: src/Headcrab.cpp [hunk 2]
```diff
@@ -6,7 +6,7 @@
 
 #include "Headcrab.h"
 #include "boost/thread.hpp"
-#include "g2log.hpp"
+#include "g3log/g3log.hpp"
```
Comment (craig-cogdill) [inline src/Headcrab.cpp]: Are we having all of these be ```#include <g3log/g3log.hpp>``` or ```#include "g3log/g3log.hpp"``` ? We should be consistent

**Comment by craig-cogdill** [comment]
:+1: after 2 comments

File: src/Headcrab.cpp (hunk 2 above)
Comment (shwetabhandare) [inline src/Headcrab.cpp]: I think it should be  #include <g3log/g3log.hpp>

**Comment by shwetabhandare** [comment]
:+1:

File: src/Crowbar.cpp (hunk 1 above)
Comment (KjellKod) [inline src/Crowbar.cpp]: Thanks. I will go through what John did and make sure I don't overwrite his changes

**Pull request merged**